docs/reagent-integration.md

```diff
@@ -0,0 +1,112 @@
+# Reagent Integration
+
+Helix can be easily integrated into a Reagent project, which can be useful if you'd like easy access to hooks or less indirection in the creation of your React components.
+
+Calling Helix components from Reagent components requires nothing special; just use the regular Helix component construction form:
+
+``` clojure
+(require '[helix.core :as  h])
```

Comment:
In general I would prefer we stay consistent with the rest of the docs when it comes to namespacing; that is, we assume `:refer [defnc $ <>]` and `helix.dom` is aliased to `d`.